libraries/AP_HAL_ChibiOS/hwdef/fmuv3/hwdef.dat

```diff
@@ -500,3 +501,9 @@ DMA_PRIORITY USART6*
 # is "ROMFS ROMFS-filename source-filename". Paths are relative to the
 # ardupilot root.
 ROMFS io_firmware.bin Tools/IO_Firmware/fmuv2_IO.bin
+
+define OSD_ENABLED ENABLED
```

Comment:
I think we'd need to make this:
 define OSD_ENABLED !HAL_MINIMIZE_FEATURES
to make it still fit on fmuv2 boards, which are derived from fmuv3.
Another approach that I'd really like to see implemented is to create a AP_OSD_Serial backend, and create a new firmware for the minimosd boards which uses a delta-protocol for screen painting. That way minimosd boards could be used on a UART with any flight controller

Reply:
thanks @tridge for looking into this. imho it still fits 1MB flash with 872kB. 
would this line:
define OSD_ENABLED !HAL_MINIMIZE_FEATURES
be added to each vehicle/config.h or does it work in hwdef.dat too?

re OSD serial backend: i personally am very happy we have finally gotten over additional minimosd firmware / fontfile handling and all respective setup-, maintenance- and power-issues. SPI OSD backend works as is without additional setup required and does not eat up a port that's usually used for other periphererals. i would not want to go back, and i wonder if it's really worth the effort to add backward compatibility for that hardware with a protocol that limits a port's protocol to a single use.

Reply:
@vierfuffzig The point about osd over serial is what different hw (not max7456/at7456) could be used as graphical backend.

Reply:
alex, it was not my intention to second-guess the potential future benefits of a serial osd backend. i‘m just not convinced about 328p/7456 type hardware, as your superb onboard osd implementation has taken us quite a bit beyond that.
the intention of this PR however merely was to optionally make use of the onboard osd‘s benefits on non-aio boards with setup-free hardware.